packages/storybook/stories/va-alert.stories.jsx

```diff
@@ -47,7 +51,7 @@ const defaultArgs = {
   'closeable': false,
   'full-width': false,
   'headline': <h3 slot="headline">Alert headline</h3>,
-  'onclose': undefined,
+  'vaClose': undefined,
```

Comment:
should this be `onVaClose`?

Reply:
I removed this because it was no longer necessary since this prop is being displayed now.